File: Entities/ResumenPedido.cs
namespace Entidades
{
    using System;
    using System.Collections.Generic;

    public class ResumenPedido
    {
        private string _proveedor;
        private bool _imprime;
        private bool _enviarEmail;
        private bool _pedir;
        private List<string> _email;
        private string _transporte;

        public string Proveedor
        {
            get => 
                this._proveedor;
            set => 
                this._proveedor = value;
        }

        public bool Imprime
        {
            get => 
                this._imprime;
            set => 
                this._imprime = value;
        }

        public bool EnviarEmail
        {
            get => 
                this._enviarEmail;
            set => 
                this._enviarEmail = value;
        }

        public bool Pedir
        {
            get => 
                this._pedir;
            set => 
                this._pedir = value;
        }

        public List<string> Email
        {
            get => 
                this._email;
            set => 
                this._email = value;
        }

        public string Transporte
        {
            get => 
                this._transporte;
            set => 
                this._transporte = value;
        }
    }
}

File: Entidades/ResumenPedido.vb
Public Class ResumenPedido

    Private _proveedor As String
    Public Property Proveedor() As String
        Get
            Return _proveedor
        End Get
        Set(ByVal value As String)
            _proveedor = value
        End Set
    End Property

    Private _imprime As Boolean
    Public Property Imprime() As Boolean
        Get
            Return _imprime
        End Get
        Set(ByVal value As Boolean)
            _imprime = value
        End Set
    End Property

    Private _enviarEmail As Boolean
    Public Property EnviarEmail() As Boolean
        Get
            Return _enviarEmail
        End Get
        Set(ByVal value As Boolean)
            _enviarEmail = value
        End Set
    End Property

    Private _pedir As Boolean
    Public Property Pedir() As Boolean
        Get
            Return _pedir
        End Get
        Set(ByVal value As Boolean)
            _pedir = value
        End Set
    End Property

    Private _email As List(Of String)
    Public Property Email() As List(Of String)
        Get
            Return _email
        End Get
        Set(ByVal value As List(Of String))
            _email = value
        End Set
    End Property

    Private _transporte As String
    Public Property Transporte() As String
        Get
            Return _transporte
        End Get
        Set(ByVal value As String)
            _transporte = value
        End Set
    End Property
End Class
